### PR TITLE
Es6 projects common modules charts doughnut

### PR DESCRIPTION
**Pull request by gidsg**
ES6 conversion.

### CODE REVIEW

File: static/src/javascripts/lib/$.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ bonzo.aug({
 const $ = (selector: ?string | ?Node, context: ?Node | ?string): bonzo =>
     bonzo(qwery(selector, context));
 
-$.create = (s: string): bonzo => bonzo(bonzo.create(s));
+$.create = (s: string | Node): bonzo => bonzo(bonzo.create(s));
```
Comment (gidsg) [inline static/src/javascripts/lib/$.js]: Turns out `bonzo.create` accepts a Node or String as per https://github.com/ded/bonzo/blob/fdfe965179754a9121c0ce3b0defec64e96934ae/bonzo.js#L1031

File: static/src/javascripts/projects/common/modules/charts/doughnut.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    const $svg = $.create(
+        '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="chart chart--doughnut"></svg>'
+    ).attr({
+        width: w,
+        height: h,
+        viewbox: `0 0 ${[w, h].join(' ')}`,
+    });
+
+    let // Segments
+    segmentAngle;
```
Comment (katebee) [inline static/src/javascripts/projects/common/modules/charts/doughnut.js]: These lines look odd, why not place the comment the `lets`, and remove the empty line on 60?

```
// segments
let segmentAngle;
let endRadius
let arc;
...
```

File: static/src/javascripts/projects/common/modules/charts/doughnut.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
+// @flow
+/*
+ * Inspired by
+ * - http://www.chartjs.org/
+ * - http://codepen.io/githiro/pen/ICfFE
+ * - https://github.com/mbostock/d3/blob/master/src/svg/arc.js
+ */
+import $ from 'lib/$';
+import bonzo from 'bonzo';
```
Comment (JonNorman) [inline static/src/javascripts/projects/common/modules/charts/doughnut.js]: If you only need `bonzo` for the types then can you import only the type to make this explicit?

File: static/src/javascripts/projects/common/modules/charts/doughnut.js [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+const translate = (v: Array<number>): string => `translate(${v.toString()})`;
+
+/**
+ * @param {Object.<string, *>} data in the format { label: string, value: number, color: HEX }
+ * @param {Object.<string, *>} o the options
+ * @return {Bonzo} SVG Element
+ */
+const Doughnut = (data: Object, o: Object): bonzo => {
+    let obj: Object = o;
+    obj = Object.assign(
```
Comment (JonNorman) [inline static/src/javascripts/projects/common/modules/charts/doughnut.js]: Could this be put into a single expression?

```js
let obj: Object = Object.assign(
        {
                percentCutout: 35,
                unit: '',
                showValues: false,
        },
        o || {}
);
```

File: static/src/javascripts/projects/common/modules/charts/doughnut.js [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+    const $svg = $.create(
+        '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="chart chart--doughnut"></svg>'
+    ).attr({
+        width: w,
+        height: h,
+        viewbox: `0 0 ${[w, h].join(' ')}`,
+    });
+
+    // Segments
```
Comment (JonNorman) [inline static/src/javascripts/projects/common/modules/charts/doughnut.js]: Could these be declared within the `data.forEach` block? They aren't referenced outside of each `forEach` iteration.

**Review by JonNorman — APPROVED** [review]
A few comments but nothing blocking IMO. Some of the variable naming is a bit suspect but nothing that you've done!

**Pull request merged**